hardhat.config.ts

```diff
@@ -63,4 +65,13 @@ export default {
       url: "https://xdai.poanetwork.dev",
     },
   },
+  namedAccounts: {
+    deployer: 0,
+    owner: {
+      // The contract deployment addresses depend on the owner address.
+      // To have the same addresses on all networks, the owner must be the same.
+      default: "0x" + "1".padStart(40, "0"),
+      hardhat: 1,
```

Comment:
What does `hardhat: 1` mean?

Reply:
It means that the the owner is the second address defined in the network configurations (`accounts[1]`). I'll add a line of comment about this.